Feature second milestone

RossielCS opened the pull request:
## This PR addresses the addition of the following features:

- A components directory.
- Implement four components: App, Display, ButtonPanel, and Button.
- The App component renders the Display and ButtonPanel components.
- The Display component accepts the result of the calculation as an incoming prop.
- The Button component accepts a button name as a prop.
- The ButtonPanel component renders the calculator panel with all the buttons.

RossielCS committed:
Add components

- Add display, button and buttonpanel
- App component renders display and buttonpanel

RossielCS committed:
Implement display and button components

RossielCS committed:
Implement buttonpanel component

RossielCS committed:
Add buttons in buttonpanel component

- Add name to each element
- Buttons divided by divs

bellom commented on src/components/ButtonPanel.js:
```diff
@@ -0,0 +1,38 @@
+
+const ButtonPanel = () => (
+  <div>
+    <div className="board-row">
+      <Button name="AC" />
+      <Button name="+/-" />
+      <Button name="%" />
+      <Button name="÷" />
+    </div>
+    <div className="board-row">
+      <Button name="7" />
+      <Button name="8" />
+      <Button name="9" />
+      <Button name="X" />
+    </div>
+    <div className="board-row">
+      <Button name="4" />
+      <Button name="5" />
+      <Button name="6" />
+      <Button name="-" />
+    </div>
+    <div className="board-row">
+      <Button name="1" />
+      <Button name="2" />
+      <Button name="3" />
+      <Button name="+" />
+    </div>
+    <div className="board-row">
+      <Button name="0" />
+      <Button name="." />
+      <Button name="=" />
+    </div>
```
### [Optional]
I think we could make use of two loops to make the Button instead of repeating ourselves. let's practice [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) code.

To do this; create an array of symbols and loop through the array using [map](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map) to create the ButtonPanel component.

bellom reviewed (approved):
## Hi @RossielCS :clap: 

##### You have done wonderfully well, you passed all checklist for this milestone :+1:
 
**Let's make your repo looks professional by applying all the 3 basic rules, kindly update your `README` file and remove unnecessary content**.
Checkout out the below link for help, feel free to follow this [template](https://github.com/microverseinc/readme-template) from pathwright

![image](https://user-images.githubusercontent.com/31897434/92685488-b4c45680-f32f-11ea-9bea-222a21c096a6.png)

https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/

**I trust you can do this before merging, 👀 on you 🙂** 

# Status: Approved 👍 

## Congratulation :+1:  
_Go ahead and merge to dev and don't forget to submit the project completion form.
Good luck with your next milestone :slightly_smiling_face:_ 

**Best,**

**Oluwaseun J. Bello**
**_Slack Handle - @sean bellom_**

RossielCS committed:
Update README

RossielCS committed:
Add uuid package to generate unique keys

- Remove buttons in buttonpanel
- Add maps in buttonpanel to dynamically create elements
